### PR TITLE
[Azquery][Readme] Edit pass

### PR DESCRIPTION
**Pull request by dlepow**
- [X ] The purpose of this PR is explained in this or a referenced issue. 
- [X ] The PR does not update generated files. Only editorial updates to Readme
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/main/CHANGELOG.md

### CODE REVIEW

**Comment by dlepow** [comment]
@sandeep-sen - This is mostly copyedits and slight rewordings for Readme consistency, as discussed. Thanks!

File: sdk/monitor/azquery/README.md [hunk 1]
```diff
@@ -1,35 +1,39 @@
 # Azure Monitor Query client module for Go
 
-The Azure Monitor Query client library is used to execute read-only queries against [Azure Monitor][azure_monitor_overview]'s two data platforms:
+The Azure Monitor Query client module is used to execute read-only queries against [Azure Monitor][azure_monitor_overview]'s two data platforms:
```
Comment (gracewilcox) [inline sdk/monitor/azquery/README.md:3]: My understanding is that "module" is the proper Go terminology.

File: sdk/monitor/azquery/README.md [hunk 2]
```diff
@@ -130,13 +135,16 @@ comments.
...
 [example_metrics_client]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/monitor/azquery#example-NewMetricsClient
+[go_samples]: (https://github.com/Azure-Samples/azure-sdk-for-go-samples)
 [kusto_query_language]: https://learn.microsoft.com/azure/data-explorer/kusto/query/
 [kusto_to_sql]: https://learn.microsoft.com/azure/data-explorer/kusto/query/sqlcheatsheet
 [log_analytics_workspace]: https://learn.microsoft.com/azure/azure-monitor/logs/log-analytics-workspace-overview
 [log_analytics_workspace_create]: https://learn.microsoft.com/azure/azure-monitor/logs/quick-create-workspace
 [logs_overview]: https://learn.microsoft.com/azure/azure-monitor/logs/data-platform-logs
 [metrics_overview]: https://learn.microsoft.com/azure/azure-monitor/essentials/data-platform-metrics
 [monitor_docs]: https://learn.microsoft.com/azure/azure-monitor/
+[monitor_rest_docs]: https://learn.microsoft.com/en-us/rest/api/monitor/
```
Comment (gracewilcox) [inline sdk/monitor/azquery/README.md:146]:
```suggestion
[monitor_rest_docs]: https://learn.microsoft.com/rest/api/monitor/
```
Can't have en-us in a link

File: sdk/monitor/azquery/README.md [hunk 3]
```diff
@@ -1,35 +1,39 @@
...
 - [Metrics][metrics_overview] - Collects numeric data from monitored resources into a time series database. Metrics are numerical values that are collected at regular intervals and describe some aspect of a system at a particular time. Metrics are lightweight and capable of supporting near real-time scenarios, making them particularly useful for alerting and fast detection of issues.
 
-[Source code][azquery_repo] | [Package (pkg.go.dev)][azquery_pkg_go] | [Product documentation][monitor_docs] | [Samples][azquery_pkg_go_samples]
+Key links:
+
+[Source code][azquery_repo] | [API reference documentation][azquery_pkg_go_docs] | [REST API documentation][monitor_rest_docs] | [Product documentation][monitor_docs] | [Samples][go_samples]
```
Comment (gracewilcox) [inline sdk/monitor/azquery/README.md:10]: Not sure if I like the changes to the links. "Key links" heading is not found in other sdks. 

Samples should link to the samples on the pkg.go website, not the general sample website. Should still keep the Package pkg.go.dev link.

Comment (dlepow) [reply sdk/monitor/azquery/README.md]: Thanks for feedback. Still converging on a common style here and we may want to consider whether the main context for the README is the pkg.go.dev site or the GitHub repo. If the former, the Package link is circular.
I removed "Key links:" heading, and restored the Package pkg.go.dev link and original "Samples" link.

**Review by gracewilcox — APPROVED** [review]
Thank you! Looks good to me.

**Comment by dlepow** [comment]
@sandeep-sen - I think this PR is ready to merge whenever it makes sense for you. Thanks!

**Pull request merged**